frontend/javascripts/oxalis/view/novel_user_experiences/welcome_toast.js

```diff
@@ -0,0 +1,72 @@
+// @flow
+import React, { useEffect } from "react";
+import { Button, notification } from "antd";
+import { useSelector } from "react-redux";
+import features from "features";
+
+function showWelcomeToast() {
+  notification.open({
+    message: (
+      <div>
+        <span className="logo" />
+        Welcome to webKnossos!
+      </div>
+    ),
+    description: (
+      <div>
+        webKnossos is a web-based platform for visualization, annotation, and sharing of large-scale
+        3D image datasets. Try out the annotation features and upload your own data with a free
+        account.
+        <div style={{ marginTop: 12, display: "flex", justifyContent: "center" }}>
```

Comment:
I think left-aligned looks better 
<img width="970" alt="Screenshot 2021-06-28 at 15 02 05" src="https://user-images.githubusercontent.com/335438/123640935-f4f2b800-d821-11eb-94dc-6a94b831471e.png">

Also, the blue looks slightly different than the blue in the "create annotation" button. Is that something you could re-use from ant css classes?

Reply:
two very good points! wasn't sure about the left aligned version first, which is why I centered it, but now I agree with you. the special blue color was the consequence of some trouble with the antd "primary" style not working at first, which I forgot about. Both issues should be fixed now:

![image](https://user-images.githubusercontent.com/2486553/123641646-c2958a80-d822-11eb-840a-513e61583ac9.png)